pymysql/util.py

```diff
@@ -17,3 +17,18 @@ def join_bytes(bs):
         for b in bs[1:]:
             rv += b
         return rv
+
+# https://dev.mysql.com/doc/internals/en/integer.html#packet-Protocol::LengthEncodedInteger
+def lenenc_int(i):
```

Comment:
lenenc-int appears in other places outside connection like https://dev.mysql.com/doc/internals/en/com-query-response.html, https://dev.mysql.com/doc/internals/en/binary-protocol-resultset.html and https://dev.mysql.com/doc/internals/en/table-map-event.html